athf/core/llm_provider.py

```
@@ -165,10 +166,10 @@ class LiteLLMProvider(LLMProvider):
     imported lazily so it is only required when this provider is actually used.
 
     Args:
-        model: A litellm-compatible model string (e.g. ``"anthropic/claude-sonnet-4-5-20250514"``).
+        model: A litellm-compatible model string (e.g. ``"anthropic/claude-sonnet-4-6"``).
     """
 
-    def __init__(self, model: str = "anthropic/claude-sonnet-4-5-20250514"):
+    def __init__(self, model: str = "anthropic/claude-sonnet-4-6"):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)